ledger/ledger-consensual/src/main/kotlin/net/corda/v5/ledger/consensual/transaction/ConsensualSignedTransactionVerifier.kt

```diff
@@ -0,0 +1,13 @@
+package net.corda.v5.ledger.consensual.transaction
+
+import net.corda.v5.base.annotations.CordaSerializable
+import net.corda.v5.base.annotations.Suspendable
+
+@CordaSerializable
```

Comment:
Why does this need to be `@CordaSerializable`? Will we store that or send it over the wire? I thought the verification in the `receiveFinality` call would be up to the local code?